### PR TITLE
[FIX]: Page Responsiveness

### LINKED ISSUE #1: Make the temporary landing page responsive

### PR DESCRIPTION
**Pull request by AyushiSrivastava11**
This PR makes the landing page responsive for mobile and tablet devices.
 - Improved the mobile view responsiveness of the Home page by updating lamp component.

Fixes #1

### CODE REVIEW

**Review by Arkapravo-Ghosh — CHANGES_REQUESTED** [review]
Need the lamp length to change dynamically based on screen size

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -4,33 +4,35 @@ import glowing_penguin from "../assets/images/glowing_penguin.svg";
...
-          >
-            We appreciate your patience but<br />something really cool is coming soon!
-          </motion.h1>
-        </LampContainer>
-        <img src={glowing_penguin}
-          alt="Glowing Penguin"
-          className="absolute right-0 bottom-[-3%]"
-        />
-      </div>
+    <div className="relative min-h-screen overflow-x-hidden overflow-y-auto">
```
Comment (Arkapravo-Ghosh) [inline src/pages/Home.tsx]: This is better I think. Try once.

```suggestion
    <div className="relative min-h-screen overflow-hidden">
```

**Review by Arkapravo-Ghosh — CHANGES_REQUESTED** [review]

**Comment by Arkapravo-Ghosh** [comment]
@captain-n3m0 Can you try looking into the requested changes?

**Comment by captain-n3m0** [comment]
> @captain-n3m0 Can you try looking into the requested changes?

Working on it.

**Pull request merged**